tests/SmartContract.spec.ts

```diff
@@ -59,6 +59,10 @@ describe('Smart Contract', () => {
             await adminContract.grantTrustedManagerPermission(user.address);
         });
 
+        test(' revoke trusted manager permission', async () => {
```

Comment:
Let's remove the space

```suggestion
        test('revoke trusted manager permission', async () => {
```